spec/models/user_spec.rb

```diff
@@ -12,6 +12,16 @@
       expect(FactoryGirl.build(:user, :email => "thisguy@@manageiq.com")).not_to be_valid
     end
 
+    it "should invalidate email addresses with uppercase letters" do
```

Comment:
This looks great overall, but I think this test belongs in the more_core_extensions gem. In moving things around earlier, I tried to separate tests for *what* was valid as belonging to the gem, and *that* the model validates as belonging here.